docs/users_guide/community_index.md

```diff
@@ -48,25 +46,67 @@ In addition to the following information, The Carpentries also maintains data on
 
 Individuals can view this information but can not edit it directly.  Please contact team@carpentries.org with questions about this information.
 
-### Instructor Training
+## Instructor Training
+
+Carpentries Instructor trainees can [log in to AMY](#logging-in) to view their checkout progress towards their Instructor badge and submit their "Get Involved" step towards certification.  More information about the [Instructor Training checkout process is in our curriculum](https://carpentries.github.io/instructor-training/checkout.html).  Log into your AMY profile and click on "Training Progress" in the top menu bar:
+
+![AMY Training Progress Menu](images/training_progress_menu.png)
+
+You may submit your "Get Involved" step here, to be evaluated by The Carpentries Instructor Training team.  The Instructor Training team will also record your status in completing the Teaching Demo and Welcome Session requirements and award your certificate upon completion of all three steps.  
+
+**Please allow 7-10 days for each of these steps to be evaluated and recorded.**
+
+### Submitting your "Get Involved" step
+
+To submit your "Get Involved" step, click on the blue "Submit a Get Involved Activity" button.  
+
+![AMY Training Progress Submit](images/training_progress_submit.png)
+
+This will take you to a page where you can select what type of activity you are submitting and provide additional information.  
```

Comment:
Suggestion to go along with https://github.com/carpentries/amy/pull/2509/files#r1291340107
```suggestion
This will take you to a page where you can select what type of activity you are submitting and provide additional information. The table below shows which information must be provided for each activity.
```